go/libkb/constants.go

```diff
@@ -657,3 +657,5 @@ const MaxTeamMembersForPairwiseMAC = 100
 
 const MaxStellarPaymentNoteLength = 500
 const MaxStellarPaymentBoxedNoteLength = 1000
+
+const ClientTriplesecVersion = 3
```

Comment:
still on 3 for now

Reply:
The problem I'm seeing is if we ever update this to 4, even if the keystretching doesn't change, we'll be setting `pwh_version=4` in the passphrase metadata, which seems like a more complicated change. Is the answer to have a `ClientTriplesecKeystretchVersion` that will continue to be 3, and two `TripleSec` instances in `g`, one for key stretching and one for encrypting/decrypting SKBs?